packages/gappa/gappa.1.3.5/opam

```diff
@@ -40,8 +40,8 @@ depends: [
 ]
 synopsis: "Tool intended for formally proving properties on numerical programs dealing with floating-point or fixed-point arithmetic"
 url {
-  src: "https://gitlab.inria.fr/gappa/gappa/-/archive/gappa-1.3.5.tar.gz"
-  checksum: "sha512=29ce59af97e6d60547a193b43538f4812ff74fb01a812cda7109855219457fa7a47f59ea39aff2a5e03fd70181e024a3296b4f48300818a81f62fd2d8629c389"
+  src: "https://gappa.gitlabpages.inria.fr/releases/gappa-1.3.5.tar.gz"
+  checksum: "sha512=60b5719e3a321df43e33045fa8f4511fc02a4218d1ae7e476e7c6ebcf90ae208832881f6eea5b99a3296dfcc3a18c7e1f4ea9dbea446fc502e14306b6975f6e6"
```

Comment:
We are trying to avoid changing the checksums. Can you get the release from the opam cache and replace the tarball at the new address? 
Caches are listed here: https://github.com/ocaml/opam-repository/wiki/FAQ#how-to-find-lost-archives-of-packages